keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/DeliveryConfigRepositoryTests.kt

```diff
@@ -702,5 +703,65 @@ abstract class DeliveryConfigRepositoryTests<T : DeliveryConfigRepository, R : R
         }
       }
     }
+
+    context("multiple delivery configs exist") {
+      before {
+        (1..5).forEach { index ->
+          val resources = setOf(
+            resource(
+              application = "${deliveryConfig.application}$index",
+              kind = parseKind("ec2/cluster@v1")
+            ),
+            resource(
+              application = "${deliveryConfig.application}$index",
+              kind = parseKind("ec2/security-group@v1")
+            )
+          )
+          val deliveryConfig = deliveryConfig.copy(
+            application = "${deliveryConfig.application}$index",
+            name = "${deliveryConfig.name}$index",
+            artifacts = setOf(
+              artifact, artifactFromBranch
+            ),
+            environments = setOf(
+              Environment(
+                name = "test",
+                resources = resources
+              ),
+              Environment(
+                name = "staging",
+                resources = resources
+              )
+            )
+          )
+          store(deliveryConfig)
+          storeResources(deliveryConfig)
+        }
+      }
+
+      test("application summaries can be retrieved successfully") {
+        val persistedDeliveryConfigs = (1..5).map { index ->
+          repository.get("${deliveryConfig.name}$index")
+        }
+
+        expectThat(repository.getApplicationSummaries())
+          .hasSize(5)
+          .containsExactlyInAnyOrder(
+            (0..4).map { index ->
+              with (persistedDeliveryConfigs[index]) {
+                ApplicationSummary(
+                  deliveryConfigName = name,
+                  application = application,
+                  serviceAccount = serviceAccount,
+                  apiVersion = apiVersion,
+                  createdAt = metadata["createdAt"] as Instant,
+                  resourceCount = 4, // 2 from test, 2 from staging
+                  isPaused = false
+                )
+              }
+            }
```

Comment:
Does this give you decent output if it fails? It feels like it could use boiling down to the thing you're actually trying to prove. It's not clear to me exactly what that is from looking here. How many results there are? The fact that each has a `createdAt` / `resourceCount` / `isPaused` value? Whether those values are correct? 

What value do you get from returning multiple results here? There's not any difference in behavior if there's one result or any number, no? They're all configured the same so I'm kind of confused what the point of this test is.

Reply:
My thinking was to try and verify that we can retrieve more than one `ApplicationSummary` and that it has the right properties (including `createdAt`, `resourceCount` and `isPaused`). Agreed that it shouldn't matter if it's one or 5. I'll make it a single object.

Reply:
I simplified the test per the above. Let me know if it looks better now.